crawl-ref/source/viewmap.cc

```diff
@@ -1044,7 +1043,7 @@ map_control_state process_map_command(command_type cmd, const map_control_state&
                 break;
             if (env.map_forgotten)
                 _unforget_map();
-            MapKnowledge *old = new MapKnowledge(env.map_knowledge);
+            map_knowledge_type *old = new map_knowledge_type(env.map_knowledge);
```

Comment:
I'm not too sure about this class rename. I know the codebase is a bit inconsistent when it comes to naming conventions, but classes are frequently PascalCase. However if I see a `_type` suffix my immediate assumption is this is probably some kind of enum so I think this name is particularly confusing

Reply:
Good point, I'd based the name on sweep_type but that seems to be a bit of an exception when it comes to names. I've changed the name back